celeryconfig.py

```diff
@@ -0,0 +1,9 @@
+from celery import Celery
+from django.conf import settings
+
+app = Celery('event_routing_backends')
+
+app.config_from_object(settings)
+
+app.autodiscover_tasks('celery_utils')
```

Comment:
@rehanedly why we need to pass int `celery_utils` here?

Reply:
@ziafazal I have removed it